### PR TITLE
MP-5718: Marketplace Order Management

### PR DESCRIPTION
**Pull request by volodymyr-bezuhlyi**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-order-management-feature-walkthrough.md [hunk 1]
```diff
@@ -1,44 +1,95 @@
...
-* Relations between modules in the Merchant Order entity:
 
-![Merchant order module relations](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Marketplace/Marketplace+and+Merchant+orders/Merchant+order+feature+overview/merchant-order-module-relations.png)
+## Domain model
+<!--
+- Domain model SHOULD contain all the entities that were adjusted or introduced by the feature.
+- All the new connections SHOULD also be shown and highlighted properly 
+- Make sure to follow the same style as in the example
+-->
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/041ca5e4-7738-47ac-a01b-4ed91a57662d.png?utm_medium=live&utm_source=confluence)
```
Comment (ihorsamusenko) [inline docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-order-management-feature-walkthrough.md:69]: This diagram is not accurate, please show real entities with real dependecies.
1. ProductOffer has no connection to Orders, but has to OrderItem
2. We don't have MarketplaceOrders but SalesOrders 
3. We don't have MerchantSalesOrderStatus
4. We don't have OmsMerchantOrderProcess

Please adjust.

File: docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+This article provides the details of how to create new MerchantOms flow
+
+<!---Help the readers to determine quickly if the HowTo matches their interests or needs. In a few sentences, summarize the main points of your HowTo. Make sure to include any critical definitions which will help readers evaluate the utility of your HowTo.-->
+
+<!---State the purpose of your HowTo. Explain how the reader will benefit from reading it. Give your reader an incentive or two to continue.-->
+ 
+<!---Inform your readers about any required knowledge, configuration, or resources they may need before stepping through your HowTo. Give links to resources that will be useful to fulfill the prerequisites.-->
+
+1. Create a new XML file in `config/Zed/oms` and call it `MarketplacePayment01.xml`
```
Comment (ihorsamusenko) [inline docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md:17]:
```suggestion
1. Create a new XML file `config/Zed/oms/MarketplacePayment01.xml`
```

File: docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+2. Add the Demo01 state machine process schema as following:
```
Comment (ihorsamusenko) [inline docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md:18]: It is not required to add another Payment method to create a Merchant oms flow.

File: docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+3. Activate the OMS process in config_default.php in config/shared by adding the name of the process `MarketplacePayment01` to the key [OmsConstants::ACTIVE_PROCESSES].
+
+```php
+$config[OmsConstants::ACTIVE_PROCESSES] = [
+	'MarketplacePayment01'
+];
+```
+4. Create Merchant state machines (To learn more about state machine creation, see [Tutorial - OMS and State Machines](https://spryker.atlassian.net/wiki/spaces/DOCS/pages/1018855468/Tutorial+-+OMS+and+State+Machines+-+Spryker+Commerce+OS+-review))
+
+`MainMerchantStateMachine.xml`
```
Comment (ihorsamusenko) [inline docs/marketplace/dev/howtos/how-to-create-a-new-merchant-oms-flow.md:48]: Please show full path, it is not clear what is the file you refer to at all.

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**